README.md

```diff
@@ -104,6 +104,21 @@ The form runner also runs in debug mode in docker compose, using the command `ya
 
 The debugger for the form-runner uses `nodemon` on port `9228` which is then exposed from the runner.
 
+## localstack
+LocalStack is a cloud service emulator that runs in a single container on your laptop or in your CI environment. With LocalStack, you can run your AWS applications or Lambdas entirely on your local machine without connecting to a remote cloud provider! Whether you are testing complex CDK applications or Terraform configurations, or just beginning to learn about AWS services, LocalStack helps speed up and simplify your testing and development workflow.
+
+To add a new aws service in localstack, edit the env variable in localstack compose config.
+```
+- SERVICES=s3,sqs
+```
+If you experience any issues with localstack setup. Try below steps to resolve
+1. Make the startup script `docker-localstack/setup-awslocal.sh` executable.
+
+        chmod u+x docker-localstack/setup-awslocal.sh
+2. Fix line ending on windows machine.
+
+        sed -i -e 's/\r$//' docker-localstack/setup-awslocal.sh
+
```

Comment:
thanks Navya, would you mind adding:
`How to test
Sample test for DLQ messages. In the localstack container terminal,

send message
awslocal sqs send-message --queue-url http://localstack:4566/000000000000/fsd-queue --message-body '{"key":"val"}'

recieve message
awslocal sqs receive-message --queue-url http://localstack:4566/000000000000/fsd-queue --attribute-names All --message-attribute-names All --max-number-of-messages 10 
trying to consume messages 3 times.

check dlq for transfered messages
awslocal sqs receive-message --queue-url http://localstack:4566/000000000000/fsd-dlq --attribute-names All --message-attribute-names All --max-number-of-messages 10 `